### PR TITLE
Add private network access

### PR DESCRIPTION
**Pull request by autonome**
Seems clearly referred to as a standalone feature in developer conversations, but also there's not much there there in BCD yet around it.

NOTE: see https://github.com/mdn/browser-compat-data/issues/25467

### CODE REVIEW

File: features/private-network-access.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name: Private Network Access
+description: The Private Network Access feature is a set of modifications to Fetch and HTML which are intended to mitigate the risks associated with unintentional exposure of devices and servers on a client’s internal network to the web at large.
+spec: https://wicg.github.io/private-network-access/
+compat_features:
+  - http.mixed-content.private_network_access
+  # This key is in the Fetch feature, maybe should move here.
```
Comment (jamesnw) [inline features/private-network-access.yml:6]:
```suggestion
  # TODO: This key is in the Fetch feature, maybe should move here.
```
Or if this is a candidate for the key being in both, what was the keyword we were going to use?

File: features/private-network-access.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+name: Private Network Access
+description: The Private Network Access feature is a set of modifications to Fetch and HTML which are intended to mitigate the risks associated with unintentional exposure of devices and servers on a client’s internal network to the web at large.
```
Comment (jamesnw) [inline features/private-network-access.yml:2]:
```suggestion
description: Private Network Access modifies Fetch and HTML to mitigate the risks associated with unintentional exposure of devices and servers on a client’s internal network to the web at large.
```

**Review by jamesnw — APPROVED** [review]
Some minor wording suggestions, but LGTM!

File: features/private-network-access.yml (hunk 2 above)
Comment (ddbeck) [reply features/private-network-access.yml:2]: The description talks about modifying other features. It's not clear how this is a feature to developers. I had a skim through the spec, so I don't really understand this thing, but I suspect we want to say something more in this direction:

```suggestion
description: The `Access-Control-Request-Private-Network: true` and `Access-Control-Allow-Private-Network: true` HTTP request and response headers establish permission for websites to make requests to internal networks, such as to routers.
```

Curiously, BCD doesn't appear to represent the request header. If we went this route, we probably ought to file a bug on BCD reporting it as missing.

File: features/private-network-access.yml (hunk 1 above)
Comment (ddbeck) [reply features/private-network-access.yml:6]: I'd say this key certainly belongs in this feature.

File: features/private-network-access.yml (hunk 2 above)
Comment (autonome) [reply features/private-network-access.yml:2]: Filed mdn/browser-compat-data/issues/25467

Comment (autonome) [reply features/private-network-access.yml]: Committed this change, which maybe is enough until BCD is sorted.

Comment (autonome) [reply features/private-network-access.yml]: Or we could leave this open and blocked until then.

Comment (ddbeck) [reply features/private-network-access.yml]: As long as the `Request.targetAddressSpace` property is representative of browser support for this (which it appears to be, at least for now) we don't have to block on the missing keys. Let's go with this.

**Review by ddbeck — APPROVED** [review]

**Pull request merged**